content/storage/mysql/docker-compose-for-mysql.md

```diff
@@ -36,21 +37,22 @@ version: '3.8'
 services:
   mysql-server:
     container_name: mysql-server
-    image: mysql:latest
+    image: mysql:8.0.26
```

Comment:
@dmakeroam Nice. It is better to specific version.

However, is it too strict to specify both minor and patch. 
How about setting `mysql:8` and we will get new compatibility minor version and patch update?

Reply:
@dmakeroam After you have replied me or taken an action on this feedback, I will merge your PR.
Thanks.

Reply:
I agree with this

Reply:
@aaronamm I agree